unfetter-discover/unfetter-ui
# Issue 1033 (Set minimum width for sidenav to avoid wrapping as much as possible)

For now, widen sidenav of baselines wizard to avoid wrapping of long capability group names.

## To test:
1. Pull this branch
2. Start a new baseline
3. Add the longest-named capability groups
4. Nothing should overlap
